terraform/environments/main.tf

```diff
@@ -3,7 +3,7 @@ variable "vault_addr" {
 }
 
 module "aws_deploy-main-ap-southeast-1" {
-  source            = "../modules/cloud/aws/deploy"
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v1.0.0"
```

Comment:
I see there is https://github.com/aeternity/terraform-aws-aenode-deploy/tree/v1.0.0

Not sure about syntax though it makes sense.